#!/usr/bin/env python
PACKAGE = "map_loader"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_offset", double_t, 0, "x_offset",    0.515, -1,   1)
gen.add("x_scale", double_t, 0, "x_scale",    100, 0,   100)
gen.add("y_offset", double_t, 0, "y_offset",    83, -1,   1)
gen.add("y_scale", double_t, 0, "y_scale",    100, 0,   100)
gen.add("theta_offset", double_t, 0, "theta_offset",    -0.29747, -3.1415926,   3.1415926)

exit(gen.generate(PACKAGE, "map_loader", "pcl_map"))